broadinstitute/orsp-pub
# Update Email Service

## Addresses
https://broadinstitute.atlassian.net/browse/CRIC-1260

## Changes
Implement new status/button for Submit to IRB

## Testing
- Update email service
- Update content when a new/existing Sample/Data Cohorts is added
- Update Subject/Body when sending approve/disapprove edit emails

---

- [ ] **Submitter**: Get a thumb from ORSP representative
